minicluster/src/main/java/org/apache/accumulo/minicluster/MiniAccumuloServerConfiguration.java

```diff
@@ -112,21 +113,34 @@ public Builder putDefaults() {
       }
 
       @Override
-      public Builder put(ServerType type, int numServers) {
-        // TODO use a constatnt
-        return put(type, "default", numServers);
+      public Builder putDefaultResourceGroup(ServerType serverType, int numServers) {
+        return put(serverType, "default", numServers);
```

Comment:
In `main`, you could reference https://github.com/apache/accumulo/blob/main/core/src/main/java/org/apache/accumulo/core/spi/scan/ScanServerSelector.java#L47 for the value of "default". In `elasticity`, this moved to Constants.java.

Reply:
Changed in 192900372d120d704f5818bd7b29fbde8680a3a4